test/services/rulesService.test.js

```diff
@@ -4,12 +4,12 @@ describe('rulesService', () => {
   test('rules service returns a rules object containing an array', async () => {
     expect.assertions(1)
     const rules = await rulesService.get()
-    expect(Array.isArray(rules.rules)).toEqual(true)
+    expect(Array.isArray(rules)).toEqual(true)
   })
 
   test('rules service object contains an array where the first element has an id of 1', async () => {
```

Comment:
Test description and assertion don't match again. Similar reason.

Reply:
Fixed